.editorconfig

```diff
@@ -0,0 +1,10 @@
+# http://EditorConfig.org
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+
```

Comment:
I added it to ensure my IDE conformed to the style of the existing code. More information regarding it and what it is used for at http://editorconfig.org/.